### ISSUE TITLE
Could you provide the pre-trained weights on ModelNet40? I have followed your details in the paper, but only got around 89% accuracy. Additionally, The data augmentation 

### ISSUE THREAD

**Issue by CH3COOK** [MenghaoGuo/PCT#10]

**Comment by CH3COOK** [comment]
In my experiments, the data augmentation step seems to have a negative effect on the accuracy in the ModelNet40 experiment. 
Have you encountered the same questions?

**Comment by MenghaoGuo** [comment]
Hi, 
I do not have the same question. Moreover, I want to know which data augmentation makes performance drop in your experiments.

**Comment by CH3COOK** [comment]
I performs the same augmentation as you mentioned in your paper: [-0.2, 0.2] translation, [-0.67, 1.5] scaling, and random input dropout. (does the random input dropout mean a uniform sampler of input points?)
